src/index.html

```diff
@@ -21,7 +21,335 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="catalog">
+    <header class="catalog__header">
+      <div class="catalog__nav-logo">
+        <a href="#homepage">
+          <img
+            src="images/logo.png"
+            alt="MOYO-logo"
+          />
+        </a>
+      </div>
+
+      <nav class="catalog__nav">
+        <ul class="catalog__nav-list">
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link catalog__nav-link--is-active"
+              href="#apple"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#samsung"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#smartphones"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#laptops & computers"
```

Comment:
The `&` character in URLs should be encoded as `&amp;`. This is an HTML entity that represents the `&` character.